src/pyscipopt/expr.pxi

```diff
@@ -129,14 +209,25 @@ cdef class Expr:
                     v = v1 + v2
                     terms[v] = terms.get(v, 0.0) + c1 * c2
             return Expr(terms)
+        elif isinstance(other, GenExpr):
+            return buildGenExprObj(self) * other
         else:
             raise NotImplementedError
 
+    def __div__(self, other):
+        ''' transforms Expr into GenExpr'''
+        divisor = buildGenExprObj(other)
+        # we can't divide by 0
+        if divisor.getOp() == Operator.const:
+            assert divisor.number != 0.0
```

Comment:
I think a Python programmer would expect a `ValueError` instead of an `AssertionError`, but I don't mind too much.

Reply:
I changed this and added a test for this. thanks